wls-admin-service/src/test/java/de/muenchen/oss/wahllokalsystem/adminservice/client/basisdaten/WahlenClientMapperTest.java

```diff
@@ -0,0 +1,91 @@
+package de.muenchen.oss.wahllokalsystem.adminservice.client.basisdaten;
+
+import de.muenchen.oss.wahllokalsystem.adminservice.eai.basisdaten.model.Farbe;
+import de.muenchen.oss.wahllokalsystem.adminservice.eai.basisdaten.model.WahlDTO;
+import de.muenchen.oss.wahllokalsystem.adminservice.service.wahlen.FarbeModel;
+import de.muenchen.oss.wahllokalsystem.adminservice.service.wahlen.WahlModel;
+import de.muenchen.oss.wahllokalsystem.adminservice.service.wahlen.WahlartModel;
+import java.time.LocalDate;
+import java.util.List;
+import lombok.val;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.mapstruct.factory.Mappers;
+
+class WahlenClientMapperTest {
+
+    private final WahlenClientMapper unitUnderTest = Mappers.getMapper(WahlenClientMapper.class);
+
+    @Nested
+    class ToModelList {
+
+        @Test
+        void should_returnNull_when_nullIsGiven() {
+            Assertions.assertThat(unitUnderTest.toModelList(null)).isNull();
+        }
+
+        @Test
+        void should_mapToListOfModel_when_givenListOfDTO() {
+            val nowDate = LocalDate.now();
+            val farbeDTO = new Farbe().r(1L).g(2L).b(3L);
+            val artDTO = WahlDTO.WahlartEnum.BAW;
+            val dtosToMap = List.of(
+                    new WahlDTO().wahlID("wahlID1").name("name1").reihenfolge(1L).waehlerverzeichnisnummer(1L).wahltag(nowDate).wahlart(artDTO).farbe(farbeDTO)
+                            .nummer("1"),
+                    new WahlDTO().wahlID("wahlID1").name("name1").reihenfolge(2L).waehlerverzeichnisnummer(1L).wahltag(nowDate).wahlart(artDTO).farbe(farbeDTO)
+                            .nummer("2"),
+                    new WahlDTO().wahlID("wahlID1").name("name1").reihenfolge(3L).waehlerverzeichnisnummer(1L).wahltag(nowDate).wahlart(artDTO).farbe(farbeDTO)
+                            .nummer("3"));
+
+            dtosToMap.forEach(wahl -> Assertions.assertThat(wahl).hasNoNullFieldsOrProperties());
```

Comment:
Macht die Vorbedingung für das Testing klarer

```suggestion
            Assertions.assertThat(dtosToMap).allSatisfy(wahl -> Assertions.assertThat(wahl).hasNoNullFieldsOrProperties());
```

Reply:
Bitte auch bei zweiter Testsuit in für `ToDtoList` übernehmen. Hatte ich leider nicht gesehe

Reply:
resolved with dfd90f8c7b69938a53c76a09c66ec349bcd33f0a